plugin.info.txt

```diff
@@ -0,0 +1,7 @@
+base   findologicxmlexport
+author Dominik Brader
+email  support@findologic.com
+date   2017-08-30
+name   findologicxmlexport plugin
+desc   Generates an XML based on the DokuWiki pages according to the FINDOLOGIC XML Scheme.
+url    https://www.findologic.com
```

Comment:
Probably makes more sense to link to the Github project.

Reply:
TODO:

- Link GitHub Project instead of `https://www.findologic.com`.